libs/core/include/erdblick/rule.h

```diff
@@ -66,6 +66,23 @@ class FeatureStyleRule
     [[nodiscard]] bool relationRecursive() const;
     [[nodiscard]] bool relationMergeTwoWay() const;
 
+    [[nodiscard]] bool hasLabel() const;
+    [[nodiscard]] std::string labelFont() const;
+    [[nodiscard]] glm::fvec4 const& labelColor() const;
+    [[nodiscard]] glm::fvec4 const& labelOutlineColor() const;
+    [[nodiscard]] glm::fvec4 const& labelBackgroundColor() const;
+    [[nodiscard]] std::pair<int, int> const& labelBackgroundPadding() const;
+    [[nodiscard]] std::string labelHorizontalOrigin() const;
```

Comment:
It is important that the std::string return values here are `const&` (except for labelText), otherwise there will be copies for each returned string.